domain/src/commonMain/kotlin/com/br/kmmdemo/network/NetworkRoutes.kt

```diff
@@ -1,10 +1,12 @@
 package com.br.kmmdemo.network
 
-object Routes {
+object NetworkRoutes {
     const val BASE_URL = "https://api.tomorrow.io/v4/weather"
     const val BASE_HOST = "api.tomorrow.io/v4/weather"
     const val FORECAST = "${BASE_URL}/forecast"
     const val API_KEY = "64JPsK8f3qpZkvaqAlToKoi0HMIv3ydf"
+
```

Comment:
Should home be in network routes or is that a navigation route that belongs in shared with other nav code?